Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -1698,8 +1698,7 @@ PartialResult WARN_UNUSED_RETURN BBQJIT::addI32Sub(Value lhs, Value rhs, Value&
         BLOCK(
             if (rhs.isConst()) {
                 // Add a negative if rhs is a constant.
-                m_jit.move(lhsLocation.asGPR(), resultLocation.asGPR());
```

Comment:
Why don't we need this move anymore? I would get it if the new line was

m_jit.add32(Imm32(rhs.asI32() == INT32_MIN ? rhs.asI32() : -rhs.asI32()), lhsLocation.asGPR(), resultLocation.asGPR());

Reply:
Same question here.